velox/type/DecimalUtil.h

```diff
@@ -271,6 +271,37 @@ class DecimalUtil {
     return overflow;
   }
 
+  /// This method is used to correct the sum result calculated using
+  /// addWithOverflow. In the addWithOverflow method, when summing two int128_t
```

Comment:
Let's add comments to addWithOverflow so we can refer to them here instead of repeating. Let's also clarify in comments for addWithOverflow that results should be post-processed using this method (adjustSumForOverflow).